### ISSUE TITLE
Complete Test Source

### ISSUE THREAD

**Issue by Will-Cooper** [SIMPLE-AstroDB/SIMPLE-db#149]
Can we find an object (or multiple) with complete information including: photometry (as much as possible); parallaxes, proper motions, radial velocities, etc?  
Would allow me to do some plot creation, for which I'll need a test object to get working.  
We could always make a dummy object but it would be nice to have something real yet well explored.

**Comment by kelle** [comment]
I would like this too. A source also with a bunch of spectral data.

**Comment by kelle** [comment]
@Will-Cooper , is this still high priority for you?  It's related to #444 .

**Comment by Will-Cooper** [comment]
Yes, this is the main blocker (bar time) on me making some fun plots like Toomre diagrams

**Comment by kelle** [comment]
I used this SQL Query to identify 1366 sources with at least one photometry measurement and a parallax and a proper motion.  We don't have any radial velocities at the moment.  [test_sources.csv](https://github.com/SIMPLE-AstroDB/SIMPLE-db/files/13822908/test_sources.csv)

That didn't narrow it down very much, but it's a start!

```SQL
SELECT  Photometry.source, Parallaxes.parallax, ProperMotions.mu_ra, ProperMotions.mu_dec, COUNT(Photometry.source)
FROM Photometry
INNER JOIN Parallaxes ON Photometry.source = Parallaxes.source
INNER JOIN ProperMotions ON Photometry.source = ProperMotions.source
GROUP BY Photometry.source, Parallaxes.source, ProperMotions.source
HAVING COUNT(Photometry.source) > 1
ORDER BY COUNT(Photometry.source) DESC
